src/pytest_springfield/README.md

```diff
@@ -0,0 +1,6 @@
+`dottedname` Module
+===================
+
+This module exists as a dummy module to help test imports of
+entity classes when using dotted-name strings for `EntityField`
+types.
```

Comment:
How do I import something from this path? Do you want me to manually inject this dir into the path?